packages/core/cms/faststore/sections.json

```diff
@@ -1913,53 +1913,6 @@
                   "default": "Apply"
                 }
               }
-            },
```

Comment:
I've decided to remove it already because, since it was merged only to feat/delivery-promise-1.2 (and not main), this removal won't be a breaking change.